client/src/components/SudoModePasswordField/SudoModePasswordField.js

```diff
@@ -0,0 +1,112 @@
+import classNames from 'classnames';
+import Button from 'components/Button/Button';
+import i18n from 'i18n';
+import Config from 'lib/Config';
+import backend from 'lib/Backend';
+import PropTypes from 'prop-types';
+import qs from 'qs';
+import React, { createRef, useState } from 'react';
+
+function SudoModePasswordField(props) {
+  const {
+    className,
+    panelPadded,
```

Comment:
The "panelPadded" functionality in [this PR](https://github.com/silverstripe/silverstripe-cms/pull/3053) is related to making the field looking pretty "inline" which is only used in this PR which I think we should close. If we close that, then the panelPadded functionality should be removed from this PR.